kratos/utilities/force_and_torque_utils.h

```diff
@@ -0,0 +1,74 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:        BSD License
+//	                Kratos default license: kratos/license.txt
+//
+//  Main Authors:   Máté Kelemen
+//
+
+#ifndef KRATOS_FORCE_AND_TORQUE_UTILS
+#define KRATOS_FORCE_AND_TORQUE_UTILS
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
+/**
+ * 
+ */
+class KRATOS_API(KRATOS_CORE) ForceAndTorqueUtils
+{
+public:
+    ///@name Operations
+    ///@{
+
+    /** Sum forces on all nodes
+     *  @param rModelPart model part containing all nodes to perform the sum on
+     *  @param rForceVariable nodal force variable to be summed up (example: REACTION)
+     */
+    static array_1d<double,3> SumForce(
+        const ModelPart& rModelPart,
+        const Variable<array_1d<double,3>>& rForceVariable);
+
+    /** Sum forces and torques on all nodes
+     *  @param rModelPart model part containing all nodes to perform the sum on
+     *  @param rForceVariable nodal force variable to be summed up (example: REACTION)
+     *  @param rTorqueVariable nodal torque variable to be summed up (example: MOMENT)
+     */
+    static std::array<array_1d<double,3>,2> SumForceAndTorque(
+        const ModelPart& rModelPart,
+        const Variable<array_1d<double,3>>& rForceVariable,
+        const Variable<array_1d<double,3>>& rTorqueVariable);
+
+    /** Total reaction force and moment exerted by the model part
+     *  @param rModelPart model part containing all nodes to perform the sum on
+     *  @param rReferencePoint reference point for computing moments
+     *  @param rForceVariable nodal force variable to be summed up (example: REACTION)
+     *  @param rTorqueVariable nodal torque variable to be summed up (example: MOMENT)
+     */
+    static std::array<array_1d<double,3>,2> ComputeReactionForceAndTorque(
```

Comment:
@rubenzorrilla `ComputeRactionForceAndTorque` is pretty much the only one that does what is meant for the new functionality. So I am a bit confused by your previous requests to split it into parts and also have the other function calls possible. I am also not sure about the name, as `REACTION` would suggest to only use that on reaction-type variables, but one could used it on `POINT_LOAD` or `POINT_MOMENT` as well (and many others).

I am also not totally sure about the naming. `SumForce` and `SumForceAndTorque` basically do what `SumHistoricalVariable` does (althout potentially one could call it on non-historical as well).

The new functionality has a very clear physical meaning, namely the reduction of nodal forces and moments to an equivalent system. Now after a search it seems to be consistently named in English "equivalent force-couple system". So I would strongly promote we sattle for a name such as `ComputeEquivalentForceCoupleSystem` or `ComputeEquivalentForceTorqueSystem`. So maybe all of us (non-native English speakers) should settle for something like this.

Reply:
@ruben to sum up I would suggest the following to wrap up this PR:

- `SumForce` internally directly interfacing to `VariableUtils`
- `SumForceAndTorque` with own implementation such that looping twice is avoided
- calling the 3rd function `ComputeEquivalentForceAndTorque`, which is consistent with the first 2 as well as seems to be in-line to what one could find as an English naming when looking for such functionality.

@matekelemen please update

An outlook for 2 PRs:

- computing the centroid as a reference point (in case none is user-defined)
- refactoring the function in `VariableUtils` to perhaps be able to avoid double looping when more variable names are given

Reply:
Totally agree. Thanks for the summary!